iwyu_output.cc

```diff
@@ -169,6 +169,17 @@ std::string PrintableUnderlyingType(const EnumDecl* enum_decl) {
   return std::string();
 }
 
+static bool FnDeclHasDefArg(const NamedDecl* decl) {
+  const auto* fn_decl = decl->getAsFunction();
```

Comment:
Let's spell out `FunctionDecl` here. I think `auto` is OK if the type name is spelled out on the right side, but not otherwise.

Reply:
I wanted to replace with `dyn_cast<FunctionDecl>` and forgot. Handling function templates is too questionable.